common/commands/src/validator/mixnet/operators/gateway/vesting_unbond_gateway.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::context::SigningClient;
+use clap::Parser;
+use log::info;
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    #[clap(long)]
+    pub gas: Option<u64>,
```

Comment:
I guess it's another recurring theme : )

Reply:
And again, removed